packages/bpk-docs/src/pages/DialogsPage/DialogsPage.js

```diff
@@ -143,6 +143,15 @@ const DialogsPage = () => (
     ]}
     components={components}
     readme={dialogReadme}
+    usageTable={{
+      dos: [
+        'Use when users are in a certain situation which needs acknowledgement.',
+        'Use when users need to confirm or make a choice.',
+      ],
+      donts: [
+        "Don't use when user need to focus on a separate task or gain context of where they are in the UI - use a modal instead.",
```

Comment:
We should change `user` to `users` here for consistency with the `dos` section.

Reply:
Done